update-versions.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 # https://eclipse.org/tycho/sitedocs/tycho-release/tycho-versions-plugin/set-version-mojo.html
-
-mvn org.eclipse.tycho:tycho-versions-plugin:set-version -DnewVersion=3.0.0-SNAPSHOT
+VERSION=$1
```

Comment:
Is this script working?
It would really be great if the version was derived from git tags. Or not? Is it good practice?